.ci/install_yq.sh

```diff
@@ -20,12 +20,9 @@ function install_yq() {
 
 	mkdir -p "${GOPATH}/bin"
 
-	# Workaround to get latest release from github (to not use github token).
-	# Get the redirection to latest release on github.
-	yq_latest_url=$(curl -Ls -o /dev/null -w %{url_effective} "https://${yq_pkg}/releases/latest")
-	# The redirected url should include the latest release version
-	# https://github.com/mikefarah/yq/releases/tag/<VERSION-HERE>
-	yq_version=$(basename "${yq_latest_url}")
+	# Stick to a specific version. Same used in
+	# runtime and osbuilder repos.
+	yq_version=2.3.0
```

Comment:
maybe we can have this at the versions.yaml of this repo

Reply:
yeah, that would be the best, but we need to install `yq` before we can actually read the yaml file.

Reply:
arf, catch 22...
@chavafg - do you know if this is a known fault (already reported) with `yq`? If not, can we make a report in the upstream yq github for it?

Reply:
@grahamwhaley, I wasn't actually sure if this was a problem with `yq` or with our yaml file, but thanks to @jodh-intel who already reported the issue to `yq` repo.